.github/workflows/build.yml

```diff
@@ -13,6 +13,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    container:
```

Comment:
This is much nicer. Thanks!

Reply:
Thanks!  Please keep in mind this is part of the prerequisite #481 so it'll disappear from this PR once that's in.  #481 is ready, but it's in draft until `zmk-docker` has been reviewed and adopted by zmkfirmware.